db/osqlblockproc.c

```diff
@@ -1279,17 +1280,14 @@ static int apply_changes(struct ireq *iq, blocksql_tran_t *tran, void *iq_tran,
     listc_init(&iq->bpfunc_lst, offsetof(bpfunc_lstnode_t, linkct));
 
     /* go through the complete list and apply all the changes */
+    if (iq->tranddl) pthread_mutex_lock(&ddl_lk);
```

Comment:
Can we have an option to enable nonblocking on this with trylock? IIRC, we used to return "schema change in progress" error.